core/src/test/java/org/elasticsearch/search/nested/SimpleNestedIT.java

```diff
@@ -56,7 +56,9 @@
 
 public class SimpleNestedIT extends ESIntegTestCase {
     public void testSimpleNested() throws Exception {
-        assertAcked(prepareCreate("test").addMapping("type1", "nested1", "type=nested").addMapping("type2", "nested1", "type=nested"));
+        assertAcked(prepareCreate("test")
+                .addMapping("type1", "nested1", "type=nested", "_all", "enabled=true")
+                .addMapping("type2", "nested1", "type=nested", "_all", "enabled=true"));
         ensureGreen();
 
         // check on no data, see it works
```

Comment:
There is an actual check, see a few lines down for:

```
// check that _all is working on nested docs
```

Which requires `_all` for the test to work.